### ISSUE TITLE
Error when installing serial version

### ISSUE THREAD

**Issue by leewujung** [usgs/geobipy#6]
I ran into issues when running `pip install -e .` for installing the serial version. The error seems to be related to `mtforward1d_fortranmodule.c`, which was not generated during build through `f2py`. I am not sure how to fix this. Any help will be much appreciated!

I am using python 3.7.1, below is the error message. Python 3.5/3.6 also produced the same error.
```sh
$ pip install -e .
Obtaining file:///Users/.../code/geobipy
    Complete output from command python setup.py egg_info:
    running egg_info
    running build_src
    build_src
    building extension "geobipy.src.classes.forwardmodelling.fdemforward1d_fortran" sources
    f2py options: []
      adding 'build/src.macosx-10.7-x86_64-3.7/build/src.macosx-10.7-x86_64-3.7/geobipy/src/classes/forwardmodelling/fortranobject.c' to sources.
      adding 'build/src.macosx-10.7-x86_64-3.7/build/src.macosx-10.7-x86_64-3.7/geobipy/src/classes/forwardmodelling' to include_dirs.
      adding 'build/src.macosx-10.7-x86_64-3.7/geobipy/src/classes/forwardmodelling/fdemforward1d_fortran-f2pywrappers2.f90' to sources.
    building extension "geobipy.src.classes.forwardmodelling.mtforward1d_fortran" sources
    f2py options: []
    f2py:> build/src.macosx-10.7-x86_64-3.7/geobipy/src/classes/forwardmodelling/mtforward1d_fortranmodule.c
    IOError: [Errno 2] No such file or directory: 'geobipy/src/classes/forwardmodelling/mtforward1D_fortran/m_mtforward1D.f90'. Skipping file "geobipy/src/classes/forwardmodelling/mtforward1D_fortran/m_mtforward1D.f90".
    error: f2py target file 'build/src.macosx-10.7-x86_64-3.7/geobipy/src/classes/forwardmodelling/mtforward1d_fortranmodule.c' not generated
    Reading fortran codes...
    Post-processing...
    Post-processing (stage 2)...
    Building modules...
   
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /Users/.../code/geobipy/
```

Thanks!

**Comment by leonfoks** [comment]
@leewujung Thanks for the bug report.  Give it another try when you have time.

**Comment by leewujung** [comment]
Thanks @leonfoks!
Just tried it and got the following:
```sh
Installing collected packages: geobipy
  Running setup.py develop for geobipy
    Complete output from command /Users/wj/miniconda3/bin/python -c "import setuptools, tokenize;__file__='/Users/wj/code_git/geobipy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" develop --no-deps:
    running develop
    running build_scripts
    running egg_info
    running build_src
    build_src
    building extension "geobipy.src.classes.forwardmodelling.fdemforward1d_fortran" sources
    f2py options: []
      adding 'build/src.macosx-10.7-x86_64-3.7/build/src.macosx-10.7-x86_64-3.7/geobipy/src/classes/forwardmodelling/fortranobject.c' to sources.
      adding 'build/src.macosx-10.7-x86_64-3.7/build/src.macosx-10.7-x86_64-3.7/geobipy/src/classes/forwardmodelling' to include_dirs.
      adding 'build/src.macosx-10.7-x86_64-3.7/geobipy/src/classes/forwardmodelling/fdemforward1d_fortran-f2pywrappers2.f90' to sources.
    build_src: building npy-pkg config files
    writing geobipy.egg-info/PKG-INFO
    writing dependency_links to geobipy.egg-info/dependency_links.txt
    writing entry points to geobipy.egg-info/entry_points.txt
    writing requirements to geobipy.egg-info/requires.txt
    writing top-level names to geobipy.egg-info/top_level.txt
    reading manifest file 'geobipy.egg-info/SOURCES.txt'
    writing manifest file 'geobipy.egg-info/SOURCES.txt'
    running build_ext
    customize UnixCCompiler
    customize UnixCCompiler using build_ext
    get_default_fcompiler: matching types: '['gnu95', 'nag', 'absoft', 'ibm', 'intel', 'gnu', 'g95', 'pg']'
    customize Gnu95FCompiler
    Could not locate executable gfortran
    Could not locate executable f95
    customize NAGFCompiler
    customize AbsoftFCompiler
    Could not locate executable f90
    Could not locate executable f77
    customize IBMFCompiler
    Could not locate executable xlf90
    Could not locate executable xlf
    customize IntelFCompiler
    Could not locate executable ifort
    Could not locate executable ifc
    customize GnuFCompiler
    Could not locate executable g77
    customize G95FCompiler
    Could not locate executable g95
    customize PGroupFCompiler
    Could not locate executable pgfortran
    don't know how to compile Fortran code on platform 'posix'
    warning: build_ext: f77_compiler=None is not available.
    
    building 'geobipy.src.classes.forwardmodelling.fdemforward1d_fortran' extension
    error: extension 'geobipy.src.classes.forwardmodelling.fdemforward1d_fortran' has Fortran sources but no Fortran compiler found
    
    ----------------------------------------
Command "/Users/wj/miniconda3/bin/python -c "import setuptools, tokenize;__file__='/Users/wj/code_git/geobipy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" develop --no-deps" failed with error code 1 in /Users/wj/code_git/geobipy/
```

**Comment by leonfoks** [comment]
Okay, it looks like f2py can't locate a Fortran compiler.
Go ahead and install a Fortran compiler, I usually use Brew, and "brew install gcc" gets me the latest version. You will need to make sure that the version of gfortran is at least 6 (I believe 6 is fine but anything later will work) since I used object oriented programming in Fortran.

Try again when it's installed.  I will add this information to the installation instructions too since I clearly missed it.

I will reopen this until you have it working.

**Comment by leewujung** [comment]
I was able to install the serial version after install gcc as you suggested via brew (it does take a while to build!) Thanks! 😀

PS. Just in case someone runs into the same `brew cleanup` permission issue re. ghostscript as I did below, I followed what's said on the last post of this [thread](https://discourse.brew.sh/t/brew-cleanup-has-not-been-run-in-30-days-running-now-at-every-brew-upgrade/4141).
```
Error: Permission denied @ unlink_internal - /usr/local/share/ghostscript/9.23/Resource/CIDFSubst/ipaexg.ttf
```
